pkg/zabbix_api.go

```diff
@@ -86,7 +71,7 @@ func (ds *ZabbixDatasource) ZabbixAPIQuery(ctx context.Context, tsdbReq *datasou
 		apiMethod := jsonQuery.Get("method").MustString()
 		apiParams := jsonQuery.Get("params")
 
-		result, err = ds.ZabbixRequest(ctx, dsInfo, apiMethod, apiParams)
+		result, err := ds.ZabbixRequest(ctx, dsInfo, apiMethod, apiParams)
```

Comment:
This change causes plugin crash because `result` is `nil`. If you use `:=`, `result` will be a new variable in `if {` scope and original `result` will be empty. I fixed it.